src/Test/Diagnostics/RemoteHost/RemoteHostPanel.xaml.cs

```diff
@@ -27,7 +27,9 @@ public RemoteHostPanel(Workspace workspace) : this()
             _workspace = workspace;
         }
 
+#pragma warning disable VSTHRD100 // Avoid async void methods
```

Comment:
:memo: Please file a bug to address these. They are not false positives.

Reply:
https://github.com/dotnet/roslyn/issues/38545